src/components/element-editor/basic-details/basic-details.tsx

```diff
@@ -37,9 +42,25 @@ export const BasicDetails = () => {
         return key in element;
     };
 
+    // =========================================================================
+    // USE EFFECTS
+    // =========================================================================
+
+    useEffect(() => {
+        updateFocusedElement(!!isDirty);
+    }, [isDirty, updateFocusedElement]);
+
+    useEffect(() => {
+        reset(element, {
+            keepDirty: true,
+            keepValues: true,
+        });
+    }, []);
```

Comment:
what are these useEffects meant for? I see the dirty state and values are still being properly set without them